### PR TITLE
Paseo Fixes

### PR DESCRIPTION
**Pull request by wilwade**
# Goal
The goal of this PR is:
- Update the Paseo spec file to the latest one
- Make Paseo Local work in place of Rococo Local

Follow ups from #1881 

## Testing Paseo
- `make build-testnet`
- `./target/debug/frequency --tmp`
- See that it syncs with the relay chain

## Testing Paseo Local
- `make build-local`
- `make start-relay`
- `make register`
- `./target/debug/frequency --chain frequency-paseo-local --alice --tmp`
- `./target/debug/frequency --chain frequency-paseo-local --bob --tmp --port 30340 --rpc-port 9950`
- `make onboard`
- See that it onboards and makes blocks!

### CODE REVIEW

File: resources/paseo.json
Comment (wilwade) [inline resources/paseo.json:1]: Pulled down from polkadot-sdk

File: resources/paseo-local.json
Comment (wilwade) [inline resources/paseo-local.json:1]: I manually generated this with a few fixes that are currently in PR for the Paseo runtime.

File: Makefile
Comment (wilwade) [inline Makefile:1]: Switch from `rococo-local` to `paseo-local`

File: README.md
Comment (wilwade) [inline README.md:1]: One typo fix and formatting

File: README.md [hunk 1]
```diff
@@ -194,10 +195,9 @@ docker run --rm -p 9944:9944 -p 30333:30333 frequencychain/instant-seal-node
 
 To stop running chain hit [Ctrl+C] in terminal where the chain was started.
 
-| **Node**                |             **Ports**             | **Explorer URL**                                                                          |
-| ----------------------- | :-------------------------------: | ----------------------------------------------------------------------------------------- |
-| Frequency Collator Node | ws and rpc:`9944`, p2p:`3033`     | [127.0.0.1:9944](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9944#/explorer) |
-
+| **Node**                |           **Ports**            | **Explorer URL**                                                                          |
+| ----------------------- | :----------------------------: | ----------------------------------------------------------------------------------------- |
+| Frequency Collator Node | ws and rpc:`9944`, p2p:`30333` | [127.0.0.1:9944](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9944#/explorer) |
```
Comment (wilwade) [inline README.md:200]: This was the typo fix. `3033` -> `30333`

File: scripts/init.sh
Comment (wilwade) [inline scripts/init.sh:1]: Swap to paseo-local

**Review by JoeCap08055 — APPROVED** [review]

**Pull request merged**